gen/generator.jl

```diff
@@ -0,0 +1,16 @@
+using Clang.Generators
```

Comment:
Can you add a readme to this directory explaining what it is for?

Reply:
Ok, I added a README, hopefully it makes sense!